### DISCUSSION TITLE
wavetables

### DISCUSSION THREAD

**Discussion by enorrmann** [bangcorrupt/hacktribe#346] [Ideas]
sample start position is modulable, but end position is fixed. If end position could follow start possition + wave cycle lenght we could have something like a wavetable osc
also, the sample start position is only triggered when a new note is triggered, so to achieve this, so the sample should be triggeres each wave cycle end
what do you think ?

**Comment by bangcorrupt** [comment]
If I can get control of loop start, end and play pointers we could have basic wavetables and granular synthesis.  I haven't dug into the DSP firmware very far yet, other than the hardware interrupts and effects presets.

For Freetribe there are already interpolated wavetables in @monome/aleph, I haven't tried them out yet.
